pkg/diag/validator/pod_test.go

```diff
@@ -861,3 +870,8 @@ type result struct {
 	iScheduledNotReady bool
 	isUnknown          bool
 }
+
+func truePtr() *bool {
```

Comment:
can use `util.BoolPtr(true)` ?

Reply:
I thought about it.  We can't because the util is defined in `pkg/skaffold/util`
1) `diag` should not depend on `skaffold`. Right thing to do would be to add pkg/diag/util  or move this to `pkg/util` so both libs can share the uitl code.
2) I didnt check but it could have round trip dependencies.